testsuite/MDAnalysisTests/utils/test_distances.py

```diff
@@ -76,153 +96,122 @@ def mindist(a, b, box):
 
         ref = mindist(a, b, box)
         val = MDAnalysis.lib.distances.distance_array(np.array([a]), np.array([b]),
-                                                      box=box, backend=self.backend)[0, 0]
+                                                      box=box, backend=backend)[0, 0]
 
         assert_almost_equal(val, ref, decimal=6,
                             err_msg="Issue 151 not correct (PBC in distance array)")
 
-class TestDistanceArray_Serial(_TestDistanceArray):
-    backend = "serial"
-
-class TestDistanceArray_OpenMP(_TestDistanceArray):
-    backend = "OpenMP"
-
-class _TestDistanceArrayDCD(TestCase):
-    backend = None
-
-    @dec.skipif(parser_not_found('DCD'),
-                'DCD parser not available. Are you using python 3?')
-    def setUp(self):
-        self.universe = MDAnalysis.Universe(PSF, DCD)
-        self.trajectory = self.universe.trajectory
-        self.ca = self.universe.select_atoms('name CA')
-        # reasonable precision so that tests succeed on 32 and 64 bit machines
-        # (the reference values were obtained on 64 bit)
-        # Example:
-        #   Items are not equal: wrong maximum distance value
-        #   ACTUAL: 52.470254967456412
-        #   DESIRED: 52.470257062419059
-        self.prec = 5
-
-    def tearDown(self):
-        del self.universe
-        del self.trajectory
-        del self.ca
+@pytest.fixture()
```

Comment:
We could make this fixture only be created once, and then shared between all tests that use it, should allow things to work faster.  The only problem with this is we trust that tests don't change the state of the fixture